src/components/form-input/README.md

```diff
@@ -416,6 +416,42 @@ On some browsers, scrolling the mousewheel while a numeric-like input is focused
 decrement the input's value. To disable this browser feature, just set the `no-wheel` prop to
 `true`.
 
+## Datalist support
+
+Datalists are a native HTML tag `<datalist>` that contains a list of `<option>` tags. By assigning
+an ID to the datalist tag, the list can be references from a text input by adding a `list` 
+attribute.
+
+This gives the input the behavior of a combo box or auto-complete, allowing existing values to be 
+chosen, or new values to be entered.
+
+```html
+<template>
+  <b-form-input list="my-list-id"></b-form-input>
+
+  <datalist id="my-list-id">
+    <option>Manual Option</option>
+    <option v-for="size in sizes">{{size}}</option>
+  </datalist>
+</template>
+
+<script>
+  export default {
+    data() {
+      return {
+        sizes: ['Small', 'Medium', 'Large', 'Extra Large']
+      }
+    }
+  }
+</script>
+
+<!-- b-form-input-datalist.vue -->
+```
+
+**Note:** Datalists work in conjunction with the browser's built in auto-complete, displaying datalist
+options first, followed by auto-complete options. To only display datalist options, set
+`autocomplete="off"`.
+
```

Comment:
It should also be mentioned that `<datalist>` is not supported in all browsers, and on some browsers it is buggy:

https://caniuse.com/#feat=datalist

And that users should not rely on `<datalist>` working.

Reply:
I was mulling over whether I should include support information before submitting, since it would eventually require removing it later as `<datalist>` became fully supported.

This settles it.

Reply:
Also note that `list` is not supported on type `password`